Source/WTF/wtf/SystemTracing.h

```diff
@@ -163,10 +163,21 @@ enum TracePointCode {
     WebXRCPFrameStartSubmissionEnd,
     WebXRCPFrameEndSubmissionStart,
     WebXRCPFrameEndSubmissionEnd,
+    DisplayLinkUpdate,
 
     GPUProcessRange = 16000,
     WakeUpAndApplyDisplayListStart,
     WakeUpAndApplyDisplayListEnd,
+
+#if USE(GLIB)
+    GLibPortRange = 20000,
+
+    DidRenderFrameStart,
+    DidRenderFrameEnd,
```

Comment:
I see this is named like the method where this is added, but that's already confusing. For the trace name I think it's better to use a more accurate name like `DidComposeFrameStart/End`

Reply:
Makes sense. I left the names from @alexgcastro variant before, but let's get it right now.

Reply:
Went for 'WaitForCompositionCompletion' (since either we don't wait here in case of explicit sync, or clientWait() on the fence).